ANTLR-version/App.g4

```diff
@@ -1,8 +1,8 @@
 // Define grammar
 grammar App;
-s : comment (s)? | if_loop (s)? | definition (s)? | NEWLINE (s)? | while_loop (s)? | function (s)? | EOF  ;
+s : comment (s)? | if_loop (s)? | definition (s)? | NEWLINE (s)? | while_loop (s)? | function (s)? | for_loop (s)? | EOF  ;
 
-while_loop: WHILE ('(')? conditional_statement (')')? ':' s END;
+while_loop: WHILE ('(')? conditional_statement ('(')? ':' s END;
```

Comment:
why did you make this change?

Reply:
i saw the same thing. We're up to date with the latest commit